app/src/ui/check-runs/ci-check-run-popover.tsx

```diff
@@ -81,6 +83,12 @@ export class CICheckRunPopover extends React.PureComponent<
   }
 
   public componentDidMount() {
+    const combinedCheck = this.props.dispatcher.tryGetCommitStatus(
```

Comment:
This actually should have been here already.. because onStatus calls setState.. which will error if the component isn't mounted yet. Before.. it just so happened it was mounted because the api call took long enough. (Now that I have set state earlier for loaders.. it was causing the error)